Comment: safe to drop "nightly" feature since nothing depends on it, and it is
 removed in the next upstream version (0.8) anyways
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -50,6 +50,6 @@
 [features]
 alloc = ["crossbeam-utils/alloc"]
 default = ["std"]
-nightly = ["crossbeam-utils/nightly"]
+#nightly = ["crossbeam-utils/nightly"]
 sanitize = []
 std = ["crossbeam-utils/std", "lazy_static"]
